packages/wonder-blocks-birthday-picker/src/components/birthday-picker.js

```diff
@@ -0,0 +1,236 @@
+                        style={{minWidth: 110}}
+                        testId="birthday-picker-month"
+                    >
+                        {moment.monthsShort().map((month, i) => (
+                            <OptionItem
+                                key={month}
+                                label={month}
+                                value={String(i)}
+                            />
+                        ))}
+                    </SingleSelect>
+                    <Strut size={Spacing.xSmall_8} />
+                    <SingleSelect
+                        // TODO(WB-1200.2): Add labels prop to handle
+                        // translations.
+                        placeholder={"Day"}
+                        onChange={this.handleDayChange}
+                        selectedValue={day}
+                        style={{minWidth: 100}}
+                        testId="birthday-picker-day"
+                    >
+                        {Array.from(Array(31)).map((_, day) => (
+                            <OptionItem
+                                key={String(day + 1)}
+                                label={String(day + 1)}
+                                value={String(day + 1)}
+                            />
+                        ))}
+                    </SingleSelect>
+                    <Strut size={Spacing.xSmall_8} />
+                    <SingleSelect
+                        // TODO(WB-1200.2): Add labels prop to handle
+                        // translations.
+                        placeholder={"Year"}
+                        onChange={this.handleYearChange}
+                        selectedValue={year}
+                        style={{minWidth: 110}}
+                        testId="birthday-picker-year"
+                    >
+                        {Array.from(Array(120)).map((_, yearOffset) => (
+                            <OptionItem
+                                key={String(CUR_YEAR - yearOffset)}
+                                label={String(CUR_YEAR - yearOffset)}
+                                value={String(CUR_YEAR - yearOffset)}
+                            />
+                        ))}
+                    </SingleSelect>
+                </View>
+                {error && (
+                    <>
+                        <Strut size={Spacing.xxxSmall_4} />
+                        <View style={{flexDirection: "row"}} role="alert">
+                            <Icon
+                                size="small"
+                                icon={icons.info}
+                                color={Color.red}
+                                style={{marginTop: 3}}
+                                aria-hidden="true"
+                            />
+                            <Strut size={Spacing.xxxSmall_4} />
+                            <Body style={{color: Color.red}}>{error}</Body>
+                        </View>
+                    </>
+                )}
```

Comment:
I prefer a `maybeRenderXxx` type function or even a new component to inline things like this, I think it makes things more readable.

Reply:
Good point! I'll do this in the next PR to avoid some conflicts.